test/commons/aria/validate-attr-value.js

```diff
@@ -1,52 +1,60 @@
 describe('aria.validateAttrValue', function() {
 	'use strict';
 
-	var orig = axe.commons.aria.lookupTable.attributes,
-		fixture = document.getElementById('fixture');
-
+	var fixture = document.getElementById('fixture');
 	var shadowSupport = axe.testUtils.shadowSupport;
 
+	beforeEach(function() {
+		axe.configure({
```

Comment:
Instead of setting this each time, I think we should put these back into the tests. A test should be self-descriptive. I get that's a bit repetitive but that's a good thing. It makes it easy to see if the test is correct.